components/output/sm2135.rst

```diff
@@ -0,0 +1,138 @@
+global ``sm2135`` hub and give it an id, and then define the
+:ref:`individual output channels <sm2135-output>`.
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    sm2135:
+      data_pin: GPIO12
+      clock_pin: GPIO14
+
+    # Individual outputs
+    output:
+      - platform: sm2135
+        id: output_red
+        channel: 0
+        max_power: 0.8
+      - platform: sm2135
+        id: output_green
+        channel: 1
+        max_power: 0.8
+      - platform: sm2135
+        id: output_blue
+        channel: 2
+        max_power: 0.8
+      - platform: sm2135
+        id: output_white
+        channel: 3
+        max_power: 0.8
+      - platform: sm2135
+        id: output_warmwhite
+        channel: 4
+        max_power: 0.8
+
+Configuration variables:
+************************
+
+-  **data_pin** (**Required**, :ref:`config-pin_schema`): The pin used for MOSI.
+-  **clock_pin** (**Required**, :ref:`config-pin_schema`): The pin which SCLK is
+   connected to.
+-  **id** (*Optional*, :ref:`config-id`): The id to use for
+   this ``sm2135`` component. Use this if you have multiple SM2135 chains
+   connected at the same time.
+
+.. _sm2135-output:
+
+Driver Output
+-------------
+
+The SM2135 output component exposes a SM2135 channel of a global
+:ref:`sm2135-component` as a float output.
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    sm2135:
+      data_pin: GPIO12
+      clock_pin: GPIO14
+
+    # Individual outputs
+    output:
+      - platform: sm2135
+        id: output_red
+        channel: 0
+        max_power: 0.8
+      - platform: sm2135
+        id: output_green
+        channel: 1
+        max_power: 0.8
+      - platform: sm2135
+        id: output_blue
+        channel: 2
+        max_power: 0.8
+      - platform: sm2135
+        id: output_white
+        channel: 3
+        max_power: 0.8
+      - platform: sm2135
+        id: output_warmwhite
+        channel: 4
+        max_power: 0.8
+
+Configuration variables:
+************************
+
+- **id** (**Required**, :ref:`config-id`): The id to use for this output component.
+- **channel** (**Required**, int): Chose the channel of the SM2135 chain of
+  this output component.
+- **sm2135_id** (*Optional*, :ref:`config-id`): Manually specify the ID of the
+  :ref:`sm2135-component`.
+  Use this if you have multiple SM2135 chains you want to use at the same time.
+- All other options from :ref:`Output <config-output>`.
+
+.. note::
+
+    The white LEDs are much brighter than the color LEDs and will fully overpower
+    the set color when the white level is set even a little bit high. You will need
+    to set the white level to 0 in order to get usable colors from this bulb.
+
+.. note::
+
+    This driver does not support enabling of both the color and the white channels
+    at the same time. Therefore, the :ref:`rgbw_color_interlock` should be set to true
+    when using this driver.
```

Comment:
Don't these 2 notes contradict each other?

The first says you can use white + rgb, but you need to turn down the white

And the second says it is not even supported...

Reply:
Ah my bad. Copied a bit too much here. I wanted to say that the white output typically is a lot brighter compared to the color output. I have updated this.